aporia.nim

```diff
@@ -2080,7 +2084,11 @@ proc initsourceViewTabs() =
   if win.globalSettings.restoreTabs and lastSession.len > 0:
     for i in 0 .. lastSession.len-1:
       var splitUp = lastSession[i].split('|')
-      var (filename, offset) = (splitUp[0], splitUp[1])
+      var filename, offset: string
```

Comment:
An unrelated fix for when the config file doesn't have the proper format for previously open files, something I was messing around with and forgot to switch branches. I'll take it out next commit.